src/commands/projects-list.ts

```diff
@@ -0,0 +1,51 @@
+import * as clc from "cli-color";
+import * as ora from "ora";
+// TODO(caot): Replace with proper import
```

Comment:
Typescript complains if I use `import * as Table from "cli-table"`, saying that `esModuleInterop`should be on

Reply:
If you run `tsc --init` in an empty dir, the default `tsconfig.json` contains `"esModuleInterop": true`, implying that it's a recommended option. @mbleigh, can we add that line to `tsconfig.json`?

Reply:
Mentioning again in a later comment